pages/expo/[slug].tsx

```diff
@@ -25,7 +25,7 @@ import { Sponsor } from '@lib/types';
 import { META_DESCRIPTION } from '@lib/constants';
 
 type Props = {
-  sponsor: Sponsor;
+  sponsor: Sponsor | null;
```

Comment:
These changes won't be necessary if we follow the other PR, since we add a `notFound` check 👍